verilog/tools/kythe/scope_resolver.h

```diff
@@ -168,7 +175,28 @@ class FlattenedScopeResolver {
   void MapSignatureToScopeOfSignature(const Signature& signature,
                                       const Signature& other_signature);
 
+  ScopeContext& GetScopeContext() { return scope_context_; }
+
  private:
+  // Return the global scope of the current scope_resolver.
+  const Scope* GetGlobalScope() const;
```

Comment:
Can this ever be null? If not, consider returning a const reference.

Reply:
It can be null in the first pass for a single file as the global will be under construction and not saved to the scopes yet.